stdlib/public/runtime/SwiftRT-ELF.cpp

```diff
@@ -34,14 +34,32 @@ DECLARE_SWIFT_SECTION(swift5_fieldmd)
 DECLARE_SWIFT_SECTION(swift5_assocty)
 DECLARE_SWIFT_SECTION(swift5_replace)
 DECLARE_SWIFT_SECTION(swift5_replac2)
+DECLARE_SWIFT_SECTION(swift5_builtin)
+DECLARE_SWIFT_SECTION(swift5_capture)
 }
 
 #undef DECLARE_SWIFT_SECTION
 
 namespace {
-static swift::MetadataSections sections{};
+static swift::MetadataSections sections{ swift::CurrentSectionMetadataVersion };
 }
 
+__asm__ (
+    ".section \".note.swift_reflection_metadata\", \"aw\" \n"
+);
+
+static
+__attribute__((used))
+__attribute__((section(".note.swift_reflection_metadata")))
+__attribute__((aligned(1)))
```

Comment:
Please use the reserved spellings (e.g. `__used__`).

Reply:
LLVM also already has portable #defines for many of these (LLVM_ATTRIBUTE_USED, etc.)

Reply:
The `LLVM_ATTRIBUTE_*` macros aren't really that useful.  This code is always built with the unix-like compiler (its part of the runtime).  Additionally, some of the macros should never be used by external projects (e.g. `LLVM_EXPORT`).  I think that consistently spelling the attributes with the reserved spelling is sufficient.

Reply:
done